plans/prepare.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+RHEL_RELEASE=$(rpm --eval %{?rhel})
+ARCH=$(uname -m)
+
+if [ $RHEL_RELEASE -eq 8 ]; then
```

Comment:
This will error out on fedora or anywhere that doesn't have the `rhel` rpm macro. Suggestion:
```bash
if [[ $RHEL_RELEASE -eq 8 ]]; then
```
(that is: double brackets, not single). Double brackets allow for undefined/empty variables.

Reply:
Also, could you add a short comment here and below explaining what is special about RHEL8, and `centos-release`, and RHEL > 8, and what each of these `dnf` commands are doing and why they're needed?